include/seastar/net/socket_defs.hh

```diff
@@ -184,3 +184,7 @@ struct hash<seastar::transport> {
 };
 
 }
+
+#if FMT_VERSION >= 90000
+template <> struct fmt::formatter<seastar::socket_address> : fmt::ostream_formatter {};
+#endif
```

Comment:
`fmt::formatter<ipv4_addr>` and `fmt::formatter<ipv6_addr>`

Reply:
ack.